.github/workflows/CI.yml

```diff
@@ -37,37 +52,49 @@ jobs:
             ${{ runner.os }}-test-
             ${{ runner.os }}-
       - uses: julia-actions/julia-buildpkg@latest
-        continue-on-error: ${{ matrix.version == 'nightly' }}
       - run: |
           git config --global user.name Tester
           git config --global user.email te@st.er
       - uses: julia-actions/julia-runtest@latest
-        continue-on-error: ${{ matrix.version == 'nightly' }}
       - uses: julia-actions/julia-processcoverage@v1
-        continue-on-error: ${{ matrix.version == 'nightly' }}
       - uses: codecov/codecov-action@v1
-        continue-on-error: ${{ matrix.version == 'nightly' }}
         with:
           file: lcov.info
-  # docs:
-  # Package need to be registered first
-  #   name: Documentation
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #     - uses: actions/checkout@v2
-  #     - uses: julia-actions/setup-julia@v1
-  #       with:
-  #         version: '1'
-  #     - run: |
-  #         git config --global user.name name
-  #         git config --global user.email email
-  #         git config --global github.user username
-  #     - run: |
-  #         julia --project=docs -e '
-  #           using Pkg;
-  #           Pkg.develop(PackageSpec(path=pwd()));
-  #           Pkg.instantiate();
-  #           include("docs/make.jl");'
-  #       env:
-  #         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  #         DOCUMENTER_KEY: ${{ secrets.DOCUMENTER_KEY }}
+
+  slack:
+    name: Notify Slack Failure
+    needs: test
+    runs-on: ubuntu-latest
+    if: github.event == 'schedule'
+    steps:
+      - uses: technote-space/workflow-conclusion-action@v2
+      - uses: voxmedia/github-action-slack-notify-build@v1
+        if: env.WORKFLOW_CONCLUSION == 'failure'
+        with:
+          channel: nightly-rse
+          status: FAILED
+          color: danger
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.RSE_SLACK_BOT_TOKEN }}
```

Comment:
before we open sourced this it was on Nightly-Dev.
You could argue that it is 
_Build and maintain scientific software packages inline with research goals._

but you would have to do so.

Reply:
are you saying it shouldn't be RSE? I might have misread the spread sheet and thought somebody marked this for RSE. I can change it back if it's not

Reply:
Fair enoguh lets change it.
Its just that it was Nightly-Dev before

Reply:
Yeah, that was me. I think this is RSE software because it largely exists to work around research limitations with normal PDMats. I also think more RSEs will be more likely to know how this interacts with other components/projects.